packages/docs/src/routes/community/components/group-link.tsx

```diff
@@ -1,14 +1,20 @@
-import { component$, JSXNode, Slot } from '@builder.io/qwik';
+import { component$, Slot, useStyles$ } from '@builder.io/qwik';
 
 export interface GroupLinkProps {
   link: string;
-  socialLogo?: JSXNode;
 }
 
+export const CSS = `
+  .docs article a.group-link {
+    color: var(--secondary-text-color);
+  }
+`
+
 export const GroupLink = component$((props: GroupLinkProps) => {
+  useStyles$(CSS);
```

Comment:
should it be global? or scoped to that componend with `useScopedStyles$`?

Reply:
Ok - will update. Thanks.

Reply:
FYI it's `useStylesScoped$`